apps/admin-server/src/pages/projects/[project]/statuses/[status]/index.tsx

```diff
@@ -0,0 +1,312 @@
+import * as React from 'react';
+import { zodResolver } from '@hookform/resolvers/zod';
+import * as z from 'zod';
+import { useForm } from 'react-hook-form';
+
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import { Input } from '@/components/ui/input';
+import { PageLayout } from '@/components/ui/page-layout';
+import { Heading } from '@/components/ui/typography';
+import { Separator } from '@/components/ui/separator';
+import { useRouter } from 'next/router';
+import useStatus from '@/hooks/use-status';
+import { useCallback, useEffect } from 'react';
+import toast from 'react-hot-toast';
+
+const formSchema = z.object({
+  name: z.string(),
+  seqnr: z.coerce.number(),
```

Comment:
Is deze niet optioneel in de db of wil je dat voor de netheid verplichten in de admin?

Reply:
Hij is niet optioneel in de db.